records_mover/records/targets/fileobj.py

```diff
@@ -74,8 +74,7 @@ def write_dfs(path_or_buf: Union[str, IO[str]]) -> int:
             move_count = write_dfs(text_fileobj)
             text_fileobj.detach()
         else:
-            with NamedTemporaryFile(prefix='mover_fileobj_target') as output_file:
-                move_count = write_dfs(output_file.name)
```

Comment:
here's the reason your unit tests broken

Reply:
ty!